training-maquetado/scss/list.scss

```diff
@@ -0,0 +1,35 @@
+@import './variables/_sizes';
+
+.book-list {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.book-list-author {
+  font-size: $text-list-small;
+  color: $grey;
+  line-height: 18px;
+  margin-top: 8px;
+}
+
+.book-list-background {
+  color: $wildsand;
+}
+
+.book-list-cover {
+  height: 200px;
+  width: 142px;
+}
+
+.book-list-item {
+  box-shadow: 4px 4px 10px 0 rgba(0, 0, 0, 0.5);
+  margin: 18px;
+  padding: 28px 28px 23px;
+}
+
+.book-list-title {
+  font-size: $text-list-medium;
+  font-weight: bold;
+  line-height: 18px;
```

Comment:
what we tend to use in projects is create a common class called `.medium-text` and you can set there the line-height and  the font size so you don't have to repeat this every time you use a font-size